File: app/src/main/java/com/pepefute/pikachu/beans/RealmPokemon.java
package com.pepefute.pikachu.beans;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

public class RealmPokemon extends RealmObject {

    @PrimaryKey
    private int id;
    private String name;
    private int height;
    private String type;
    private int weight;
    private String sprite;

    public RealmPokemon() {
    }

    public RealmPokemon(int id, String name, int height, String type, int weight, String sprite) {
        this.id = id;
        this.name = name;
        this.height = height;
        this.type = type;
        this.weight = weight;
        this.sprite = sprite;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getHeight() {
        return height;
    }

    public void setHeight(int height) {
        this.height = height;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public int getWeight() {
        return weight;
    }

    public void setWeight(int weight) {
        this.weight = weight;
    }

    public String getSprite() {
        return sprite;
    }

    public void setSprite(String sprite) {
        this.sprite = sprite;
    }
}

File: app/src/main/java/com/pepefute/pikachu/beans/RealmPokemon.kt
package com.pepefute.pikachu.beans

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey

open class RealmPokemon : RealmObject {

    @PrimaryKey
    var id: Int = 0
    var name: String? = null
    var height: Int = 0
    var type: String? = null
    var weight: Int = 0
    var sprite: String? = null

    constructor()

    constructor(id: Int, name: String, height: Int, type: String, weight: Int, sprite: String) {
        this.id = id
        this.name = name
        this.height = height
        this.type = type
        this.weight = weight
        this.sprite = sprite
    }
}
